### ISSUE TITLE
PUT Transaction endpoint should error if trying to unlock token before time

### ISSUE THREAD

**Issue by shaktals** [0chain/0chain#478]
Now the PUT `[miner]/v1/transaction/put` endpoint returns a 200 success when trying to unlock a token, even if it's locked time has not completed yet. I think it should error. Check #471 for context.

**Comment by shaktals** [comment]
As Piers just explained in the above mentioned issue, the request actually succeeds, but the action is postponed. It'd be nice to let the user know about that.

We could even display that info on the frontend maybe. We'd have to update the `getUserPools` endpoint too? (I quickly tried to find the docs for it but couldn't)

**Comment by Sriep** [comment]
We could change `zbox` to return a friedly message about maybe having to wait, althogh I am not sure that would help your use case.

**Comment by shaktals** [comment]
Yeah, on `block-explorer` we are actually preventing the user from trying to unlock a token before the lock time is complete. However, for any other clients consuming the API, this could be a source of confusion. I guess it'd just be a small improvement to make the API a little friendlier.

**Comment by stewartie4** [comment]
This is more of a consequence of our separation of concerns design right? - when you send a transaction, the processing is asynchronous so you've no idea whether the request succeeded or not. The 200 response is simply saying I have accepted the resource you have PUT, the processing happens later by another component.
That's why you have to confirm the transaction on the sharder after sending it

**Comment by shaktals** [comment]
Agreed, I see no problem with the `200` response. Just thought we could maybe include a piece of info on the response, along the lines of: `Unlock scheduled to be processed after lock time is complete` or something.

A small improvement, not essential. 
If it doesn't resonate with you guys, we can just close this :+1:

**Comment by Sriep** [comment]
Giving an informative response to the user should be the job of the tool sending the transaction. `zbox` for example. Closing as in the wrong project.
